Engine/Server/Src/GameEngine/Application/Application.cpp

```diff
@@ -11,73 +11,76 @@
 #include <windows.h>
 #endif
 
+GameEngine::Application::Application()
+{
+    _server = ABINetwork::createServer(100);
+    _nbRoom = 0;
+}
+
+void GameEngine::Application::run()
+{
+    while (true) {
```

Comment:
😂 nice one

Reply:
For the moment it's like this, but in the future I will create a signal catcher for SIGINT call to close the application.